src/common/processUtils.ts

```diff
@@ -32,12 +32,12 @@ export const reportError = (error: string) => {
  * to be able to track the source of the error 
  */
 export const evaluate: (
-    page: puppeteer.Page,
+    element: puppeteer.Page | puppeteer.ElementHandle<Element>,
```

Comment:
Do you mind putting the `any` keyword here:
```ts
// here:
element: puppeteer.Page | puppeteer.ElementHandle<any>,
// instead of:
return await (element as any).evaluate(pageFunction, ...args);
```

Reply:
not at all :+1: